client/src/components/Home/index.tsx

```diff
@@ -391,6 +407,8 @@ export default class Home extends React.Component<any, IHomeState> {
             defaultValue={template && template.id || ''}
             lineDirection="center"
             placeholder="Choose an ID for the dashboard (will be used in the url)"
+            error={error}
```

Comment:
The toast alert is ok for general notifications, but not great UX for form validation.
![screen shot 2017-08-17 at 11 19 53](https://user-images.githubusercontent.com/1880480/29407974-4ce5fbae-833e-11e7-8598-ef081804aaea.png)